core/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mapzen.core">
+    package="com.mapzen">
```

Comment:
I thought it might make more sense to have a generic package name as we move public facing classes like `MapzenMap` into `core`. I wanted the existence of the core module to be as transparent as possible. What are your thoughts on this?